makefile

```diff
@@ -33,3 +33,10 @@ bin/dfmt: $(SRC)
 
 debug: $(SRC)
 	$(DC) $(DMD_DEBUG_FLAGS) $^ -ofbin/dfmt
+
+pkg: dmd
```

Comment:
I would use `bin/dfmt` as dependency instead, that `dmd` target looks super weird, I would expect for `dmd` to be built for a target named like that :D. And also this is what you really want, if for some reason `make dmd` needs to do something else, you don't really care about that.